src/test/tools/obj_verify/README

```diff
@@ -0,0 +1,25 @@
+Persistent Memory Development Kit
+
+This is src/test/tools/obj_verify/README.
+
+This directory contains a simple tool 'obj_verify' for verifying pmemobj pools.
+
+'obj_verify' has three modes of operation:
+  1) 'c' - create and verify a pmemobj pool
+  2) 'v' - verify a pmemobj pool
+  3) 'd' - verify a pmemobj pool and print debug messages
```

Comment:
Why c=create&verify? c should be just 'create' and v just 'verify'. d should be a modifier for verify, not a separate mode.

You wouldn't have to explain that in the next paragraph.